tasks/job.yml

```diff
@@ -14,6 +14,7 @@
     storage_domain: "{{ cohesity_protection.storage_domain | default('DefaultStorageDomain') }}"
     delete_backups: "{{ cohesity_protection.delete_backups | default(False) }}"
     cancel_active: "{{ cohesity_protection.cancel_active | default(False) }}"
+    remove_existing: "{{ cohesity_protection.remove_existing | default(True) }}"
```

Comment:
what is current behavior ? lets default to that

Reply:
Currently, updating VMware job will replace available VMs in the job. So flag is to set to true, will replace all existing vms.